### PR TITLE
increase storage retention to 30days

### PR DESCRIPTION
**Pull request by tanvigour**
Signed-off-by: tanvigour <tanvi.gour@gmail.com>

# Description

Change prometheus storage retention time to a month so that we get data for entire month for longhaul availability

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by tanvigour** [comment]
@halspang : please review when you get a chance

**Review by halspang — CHANGES_REQUESTED** [review]
Should we do this for the regular longhaul as well?

File: .github/workflows/dapr-longhaul-nightly.yml [hunk 1]
```diff
@@ -93,8 +93,7 @@ jobs:
       # Setup monitoring
       - name: Create monitoring namespace if it doesn't exist
         run: |
-          kubectl get namespace | grep ${{ env.DAPR_MONITORING }} || kubectl create namespace ${{ env.DAPR_MONITORING }}
-      - name: check if grafana already exists
+          kubectl get namespace | grep ${{ env.DAPR_MONITORING }} || kubectl create namespace ${{ env.DAPR_MONITORING }} -set server.retention="30d"
```
[the remark this answers is not in the document]
Comment (tanvigour) [reply .github/workflows/dapr-longhaul-nightly.yml:96]: Thanks for catching this, the location i added to in the workflow wasn't right, it should be during dapr-prom install, you can see the change to 30h in dapr-prom-prometheus-server pod after that

**Comment by tanvigour** [comment]
@halspang : please review

**Pull request merged**